jdaviz/core/custom_traitlets.py

```diff
@@ -0,0 +1,26 @@
+from traitlets import Int, Float
+
+
+class HandleEmptyMixin(object):
```

Comment:
Nitpick: No need to inherit from object in Python 3. There is no old-style class anymore.

```suggestion
class HandleEmptyMixin:
```